qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationPage.java

```diff
@@ -37,6 +38,11 @@ public ManageUserPage clickOnManageUsers() {
 		findElement(MANAGE_USERS).click();
 		return new ManageUserPage(this);
 	}
+	
+	public AdministrationManageRolesPage clickOnManageRoles(){
```

Comment:
what do you thin of `clickOnManageRoles()` vs `clickOnManageRolesLink()` vs `goToManageRolesPage()`?